packages/lib/passport-interface/src/PassportPopup/react.ts

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { receiveZupassPopupMessage, zupassPopupSetup } from "./core";
+
+/**
+ * React hook that listens for PCDs and PendingPCDs from a Zupass popup window
+ * using message passing and event listeners.
```

Comment:
nit but `using message passing and event listeners` is an implementation detail the reader probably doesn't really need to know.